service/scheduler/quartz/src/main/java/org/eclipse/kapua/service/scheduler/trigger/quartz/TriggerServiceImpl.java

```diff
@@ -57,6 +59,7 @@
         implements TriggerService {
 
     private static final Domain SCHEDULER_DOMAIN = new SchedulerDomain();
+    KapuaQuery<Trigger> queryByJobId;
```

Comment:
This MUST be removed. 
Service classes are a singleton. 
Relying on the value set here is not suitable since this a value shared across the whole JVM.